config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -76,7 +76,7 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --cluster=gce-scale-cluster
-      - --env=HEAPSTER_MACHINE_TYPE=n1-standard-32
+      - --env=HEAPSTER_MACHINE_TYPE=e2-standard-32
```

Comment:
Let's not migrate those large ones at this point.

Reply:
I think that the large ones were already migrated in https://github.com/kubernetes/test-infra/pull/19594 and https://github.com/kubernetes/test-infra/pull/19603 but we forgot to change this env var appropriately in 5k performance job, hence my change here.

Shall I include this amendment in a separate PR?

Reply:
Leaving it as it is after offline discussion.